### ISSUE TITLE
About the 0 bytes bug

### ISSUE THREAD

**Issue by santisy** [CharlesShang/FastMaskRCNN#11]
I checked the _mask_targets_' shape, it seems that after being prepossessed by __filter_negative_samples_, its first dimension became 0, which is corresponding to the problem reported in the community.

**Comment by CharlesShang** [comment]
Thank you. 
Saved my day. 
Seems cross_entropy loss function does not take 0 size tensors as inputs.

**Comment by santisy** [comment]
@CharlesShang 
I didn't figure it out why the labels will be full `-1` after `mask_encoder` while many `1` by `anchor_encoder`, isn't it that the two functions all use the same `gt_boxes` and `rois`(the -1  is only triggered by `fg_threshold` overlap check)? Maybe there is some bugs in `mask_encoder`.
